department-of-veterans-affairs/caseflow
# Added logic to prevent adding tasks to a dispatched appeal

Resolves #15363 

### Description
Adds logic to prevent a ChangeHearingRequestTask from being created when visiting the case details page of a dispatched appeal.

### Acceptance Criteria

### Testing Plan
1. First open a rails console and locate a dispatched LegacyAppeal: `id=LegacyAppeal.all.select{|x| !x.active?}.first`
1. Next login as user `BVASYELLOW`
1. Visit the case details page for that appeal: `http://localhost:3000/queue/appeals/{id}`
1. Ensure that you do not get a dropdown for the change hearing request type
